packages/node-integration-tests/package.json

```diff
@@ -13,7 +13,6 @@
     "lint:eslint": "eslint . --cache --cache-location '../../eslintcache/' --format stylish",
     "lint:prettier": "prettier --check \"{suites,utils}/**/*.ts\"",
     "type-check": "tsc",
-    "pretest": "run-s --silent prisma:init",
```

Comment:
How does this relate to the PII change?

Reply:
It doesn't. Accidentally commited that change and removed it in [ed3ed28](https://github.com/getsentry/sentry-javascript/pull/5363/commits/ed3ed2879d7cbc118bd22ca79245db9ca8f9634c).

(I removed this line locally because that's how I can get the node integration tests to work locally on my computer)

Reply:
Ah, got it.

FYI, if you start up docker before running the tests, you don't need to comment it out. Somewhere on some branch that I now can't seem to find, I started working on trying to make it actually throw a warning in that case, but ended up getting sidetracked.

Reply:
Ah makes sense. I'll give this a try next time, thanks 👍